src/schemas/v3.js

```diff
@@ -0,0 +1,1827 @@
+    allowNull: false
+  }, {
+    name: 'server_updated_at',
+    type: 'timestamp',
+    allowNull: false
+  }, {
+    name: 'record_index_text',
+    type: 'string'
+  }, {
+    name: 'record_index',
+    type: 'fts'
+  }, {
+    name: 'geometry',
+    type: 'geometry'
+  }, {
+    name: 'altitude',
+    type: 'double'
+  }, {
+    name: 'speed',
+    type: 'double'
+  }, {
+    name: 'course',
+    type: 'double'
+  }, {
+    name: 'horizontal_accuracy',
+    type: 'double'
+  }, {
+    name: 'vertical_accuracy',
+    type: 'double'
+  }, {
+    name: 'form_values',
+    type: 'text'
+  }, {
+    name: 'changeset_id',
+    type: 'integer'
+  }, {
+    name: 'changeset_resource_id',
+    type: 'string'
+  }, {
+    name: 'title',
+    type: 'string'
+  }, {
+    name: 'created_latitude',
+    type: 'double'
+  }, {
+    name: 'created_longitude',
+    type: 'double'
+  }, {
+    name: 'created_geometry',
+    type: 'geometry'
+  }, {
+    name: 'created_altitude',
+    type: 'double'
+  }, {
+    name: 'created_horizontal_accuracy',
+    type: 'double'
+  }, {
+    name: 'updated_latitude',
+    type: 'double'
+  }, {
+    name: 'updated_longitude',
+    type: 'double'
+  }, {
+    name: 'updated_geometry',
+    type: 'geometry'
+  }, {
+    name: 'updated_altitude',
+    type: 'double'
+  }, {
+    name: 'updated_horizontal_accuracy',
+    type: 'double'
+  }, {
+    name: 'created_duration',
+    type: 'integer'
+  }, {
+    name: 'updated_duration',
+    type: 'integer'
+  }, {
+    name: 'edited_duration',
+    type: 'integer'
+  }
+];
+
+Schema.systemValuesTableColumns = [
+  {
+    name: 'id',
+    type: 'pk'
+  }, {
+    name: 'record_id',
+    type: 'integer',
+    allowNull: false
+  }, {
+    name: 'record_resource_id',
+    type: 'string',
+    length: 100
+  }, {
+    name: 'parent_resource_id',
+    type: 'string',
+    length: 100
+  }, {
+    name: 'key',
+    type: 'string',
+    allowNull: false,
+    length: 100
+  }, {
+    name: 'text_value',
+    type: 'string',
+    length: 500
+  }, {
+    name: 'number_value',
+    type: 'double'
+  }
+];
+
+Schema.systemRepeatableTableColumns = [
+  {
+    name: 'id',
+    type: 'pk'
+  }, {
+    name: 'resource_id',
+    type: 'string',
+    allowNull: false
+  }, {
+    name: 'record_id',
+    type: 'integer',
+    allowNull: false
+  }, {
+    name: 'record_resource_id',
+    type: 'string',
+    allowNull: false
+  }, {
+    name: 'parent_resource_id',
+    type: 'string'
+  }, {
+    name: 'record_record_key',
+    type: 'string'
+  }, {
+    name: 'record_record_sequence',
+    type: 'integer'
+  }, {
+    name: 'record_project_id',
+    type: 'integer'
+  }, {
+    name: 'record_project_resource_id',
+    type: 'string'
+  }, {
+    name: 'record_assigned_to_id',
+    type: 'integer'
+  }, {
+    name: 'record_assigned_to_resource_id',
+    type: 'string'
+  }, {
+    name: 'record_status',
+    type: 'string'
+  }, {
+    name: 'index', // TODO(zhm) make this work in the app
```

Comment:
I realize this was probably a copy paste, but is this TODO needed?

Reply:
Good question . . . I didn't want the v2 and v3 schemas to be different in ways that _don't_ reflect the changes being introduced. I'd say, if we want to clean up the comments, we do it in a different ticket??

Reply:
fine with me